Libraries/Network/XMLHttpRequestBase.js

```diff
@@ -66,6 +66,12 @@ class XMLHttpRequestBase {
     this._url = null;
     this._aborted = false;
   }
+  
+  static get UNSENT() { return 0 }
```

Comment:
Please use semicolons and adhere to the coding conventions - look around to infer what they are